linux/LinuxProcessList.c

```diff
@@ -2101,16 +2101,11 @@ static void scanCPUFrequencyFromCPUinfo(LinuxProcessList* this) {
       if (fgets(buffer, PROC_LINE_LENGTH, file) == NULL)
          break;
 
-      if (
-         (sscanf(buffer, "processor : %d", &cpuid) == 1) ||
-         (sscanf(buffer, "processor: %d", &cpuid) == 1)
-      ) {
+      if (sscanf(buffer, "processor : %d", &cpuid) == 1) {
```

Comment:
How about also remove the space right before the `%d` and `%lf` markers? They are unnecessary, too.
```suggestion
      if (sscanf(buffer, "processor :%d", &cpuid) == 1) {
```
`%d` and `%lf` formats, like `strtol` and `strtod` functions, skip leading whitespaces in the input.